omod/src/main/webapp/resources/scripts/drugOrders.js

```diff
@@ -199,7 +199,8 @@ angular.module('drugOrders', ['orderService', 'encounterService', 'uicommons.fil
                 var encounterContext = {
                     patient: config.patient,
                     encounterType: config.drugOrderEncounterType,
-                    location: null // TODO
+                    location: null, // TODO
+                    encounterRole: config.encounterRole
```

Comment:
I thought you would replace the above with EncounterProvider which is = (provider & encounterRole)